feat(imgproc): implement adjust_brightness

AnilKumarSingh9856 opened the pull request:
Closes #171

This PR adds the `adjust_brightness` function to the `kornia-imgproc` crate, as requested in the issue and discussed on Discord.

Changes:
* Implements `pub fn adjust_brightness` in `crates/kornia-imgproc/src/enhance.rs`.
* The logic is ported from the reference Python implementation, performing an additive operation and clamping the result.
* Adds three new tests (`test_adjust_brightness_normal`, `_clamping`, `_no_clip`) to verify the new function.

edgarriba commented on crates/kornia-imgproc/src/enhance.rs:
```diff
@@ -60,10 +61,74 @@ where
+    T: Float + std::fmt::Debug + Send + Sync + Copy,
+{
+    if src.size() != dst.size() {
+        // FIX: Use cols() and rows() for consistency
```
no prompted comments!

edgarriba commented on crates/kornia-imgproc/src/enhance.rs:
```diff
@@ -60,10 +61,74 @@ where
+        parallel::par_iter_rows_val(src, dst, |&src_pixel, dst_pixel| {
+            let val = src_pixel + factor;
+            // Manual clamp logic
+            *dst_pixel = if val < zero {
```
why not just using builtin clamp?

AnilKumarSingh9856 commented:
You're absolutely right, thanks! I've pushed a new commit to fix these:

  1.   Removed the // FIX: comment.
  3.   Replaced the manual if/else logic with val.max(zero).min(one) which is much cleaner and uses the Float trait's built-in methods.

AnilKumarSingh9856 commented:
Done! Switched to val.clamp(zero, one). That's much cleaner and should fix the Clippy failure. Thanks for the catch!
